app/views/teams/show.html.erb

```diff
@@ -3,6 +3,52 @@
     <h1><%= i @team.team_type.icon %> <%= @team.identifier_name %></h1>
   </div>
   <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <div style="margin-bottom: 20px;">
+          <h2>People</h2>
+          <ul>
+            <% @team.people.each do |person| %>
+              <li><%= link_to person.display_name, person %></li>
+            <% end %>
+          </ul>
+        </div>
+        <div style="margin-bottom: 20px;">
+          <h2>Badges</h2>
+          <ul>
+            <% @team.badges.each do |badge| %>
```

Comment:
There's a potential for there to be quite a few of these too, so I'd recommend capping the number you're displaying and doing them in rows & columns.